src/main/java/com/ubc/cpsc304/hotelify/entity/Review.java

```diff
@@ -0,0 +1,27 @@
+package com.ubc.cpsc304.hotelify.entity;
+
+import javax.persistence.*;
+
+import lombok.Data;
+
+@Data
+@Entity
+public class Review {
+
+    @Id
+    @Column(nullable = false, updatable = false, unique = true)
+    private Long review_id;
+
+    @Column(nullable = false, columnDefinition = "VARCHAR(255) DEFAULT ''")
+    private String username;
+
+    @Column(nullable = false)
+    private Long hotel_id;
```

Comment:
this review has a many-to-many relation with hotel, so we need to join a table here. I can show you how to do it when scott merged his hotel stuff

Reply:
I have some troubles adding @ManyToOne...IDK why